src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rotate-animation($duration, $timing-function) {
+  animation: hand-move $duration $timing-function infinite;
```

Comment:
The animation name 'hand-move' should match the mixin name 'rotate-animation' to keep consistency and improve readability. Consider renaming the keyframes to match the mixin name.